gradle-fury-validation/build.gradle

```diff
@@ -14,6 +14,7 @@ dependencies{
     compile group: 'commons-cli', name: 'commons-cli', version: '1.3'
 
 
+    compile group: 'org.jsoup', name: 'jsoup', version: '1.9.2'
```

Comment:
Methinks this is a testCompile dependency.  Might want to also property-ize as well.